src/Reporting/src/App.Metrics.Formatting.StatsD/Internal/StatsDSyntax.cs

```diff
@@ -42,7 +45,7 @@ public class StatsDSyntax
                 { AppMetricsConstants.Pack.GaugeMetricTypeValue, Gauge },
                 { AppMetricsConstants.Pack.CounterMetricTypeValue, Count },
                 { AppMetricsConstants.Pack.HistogramMetricTypeValue, Histogram },
-                { AppMetricsConstants.Pack.MeterMetricTypeValue, Meter },
+                { AppMetricsConstants.Pack.MeterMetricTypeValue, Count },
```

Comment:
Where does the actual `Meter` value get pulled inside the DogStatsD / StatsD formatting?

Reply:
It is done here: https://github.com/AppMetrics/AppMetrics/blob/features/4.2.0/src/Reporting/src/App.Metrics.Formatting.StatsD/Internal/StatsDPointSampler.cs#L77

Reply:
Count is by default sampled by StatsD while Meter isn't, I have to modify that now, don't merge yet.

Reply:
LGTM